packages/cli/src/commands/build.ts

```diff
@@ -1,7 +1,9 @@
 import {Command, flags} from '@oclif/command'
 import {build} from '@blitzjs/server'
+import {runPrismaGeneration} from './db'
 
 export class Build extends Command {
+  // eslint-disable-next-line import/no-default-export
```

Comment:
Any reason for this?

Reply:
Ah.Okay. it might occurred while i was resolving merge conflict. shall i remove this?